identity-account-storage/src/error.rs

```diff
@@ -12,6 +12,9 @@ pub enum Error {
   /// Caused by errors from the [identity_core] crate.
   #[error(transparent)]
   CoreError(#[from] identity_core::Error),
+  /// Caused by errors from the [crypto] crate.
+  #[error(transparent)]
+  CryptoError(#[from] crypto::error::Error),
```

Comment:
I might be wrong, but I was not able to find any reference to this variant in my IDE. If it is not needed then it can be removed. Regardless of whether it is actually in use or not I have a few concerns about the value of this variant.  It does not describe the context in which this error from an external crate occurred. Also not on the documentation of the variant hence it becomes very hard for those that want to handle errors of this type to understand when it may occur unless they are very familiar with the internals of this crate. Furthermore this is also a variant of `identity_core::Error` which is wrapped in the `CoreError` variant of this error enum. By exposing the error from the crypto crate we are assuming some consumers of identity.rs may want to handle errors of this type, but when it can appear without context in multiple places this task becomes increasingly hard.